Lib9c/Action/DPoS/Model/ProposerInfo.cs

```diff
@@ -0,0 +1,31 @@
+﻿using Bencodex.Types;
+using Libplanet.Crypto;
+using Nekoyume.Action.DPoS.Util;
+
+namespace Nekoyume.Action.DPoS.Model
+{
+    public class ProposerInfo
```

Comment:
```suggestion
    public class ProposerInfo : IBencodable
```
I'm not sure, but shouldn't `ProposerInfo` implement `IBencodable`?